src/renderable/sprite.js

```diff
@@ -560,8 +584,7 @@
 
             // frame offset in the texture/atlas
             var frame_offset = frame.offset;
-            var g_offset = this.offset;
-
+            var g_offset = new me.Vector2d()
```

Comment:
definitely not good here, as now me.Sprite is creating a new Vector object at each frame !

Reply:
also this.offset is global offset applied to any frame, if you redefined here g_offset, then this is a feature regression.